tests/rustdoc-ui/intra-doc/issue-108653-associated-items-6.stderr

```diff
@@ -0,0 +1,22 @@
+error: `u32::MAX` is both an associated constant and a builtin type
+  --> $DIR/issue-108653-associated-items-6.rs:4:7
+   |
+LL | /// [`u32::MAX`]
+   |       ^^^^^^^^ ambiguous link
+   |
+note: the lint level is defined here
+  --> $DIR/issue-108653-associated-items-6.rs:1:9
+   |
+LL | #![deny(rustdoc::broken_intra_doc_links)]
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+help: to link to the associated constant, prefix with `const@`
+   |
+LL | /// [`const@u32::MAX`]
+   |       ++++++
+help: to link to the builtin type, prefix with `prim@`
+   |
+LL | /// [`prim@u32::MAX`]
+   |       +++++
```

Comment:
This still doesn't seem right? One of these is a constant in a module, and the other is an associated constant to the `u32` primitive type (I think the error has those backwards as well and thinks the one in the module is an associated constant).

Reply:
Ah interesting: it considers `MAX` as a primitive from the module and as an associated constant of `u32`. The naming isn't great though...

Reply:
Well I renamed it "primitive type" so that fixes this issue too. 😆